src/main/include/commands/ScoreAmpCommand.h

```diff
@@ -1,68 +1,34 @@
 #pragma once
 
-#include <frc2/command/Command.h>
-#include <frc2/command/CommandHelper.h>
-
-#include "subsystems/ScoringSubsystem.h"
-#include "subsystems/IntakeSubsystem.h"
-
-class ScoreAmp : public frc2::CommandHelper<frc2::Command, ScoreAmp> {
-    public:
-
-        enum class ScoreState {
-            FlywheelRamp,
-            Feeding,
-            Shooting,
-        };
-
-        explicit ScoreAmp(ScoringSubsystem* subsystem, IntakeSubsystem* intk) 
-            : m_score{subsystem}, m_intake{intk}, isFinished{false} {
-                AddRequirements({m_intake, m_score});
-            }
-
-        void Initialize() override {
-            if (!m_intake->NotePresent()) {
-                isFinished = true;
-                return;
-            }
-            m_state = ScoreState::FlywheelRamp;
-            m_score->StartScoringRamp(ScoringDirection::AmpSide);
-            isFinished = false;
+#include "BaseScoreCommand.h"
+
+class ScoreAmp : public BaseScoreCommand {
+ public:
+  explicit ScoreAmp(ScoringSubsystem* subsystem, IntakeSubsystem* intk)
+      : BaseScoreCommand(subsystem, intk, ScoringDirection::AmpSide) {}
+
+  void Execute() override {
```

Comment:
Why can't `Execute` be moved into the base?

Reply:
It can be, I’ll fix that